test/functional/ui/popupmenu_spec.lua

```diff
@@ -2103,6 +2107,104 @@ describe('builtin popupmenu', function()
             ]],
           })
         end
+        feed('<C-E><ESC>')
+
+        -- avoid modified original info text
+        command('call Append_multipe()')
```

Comment:
If I'm trying to understand this test in the future, it's hard to see what this new part is testing. Is there a clearer way to describe it?

Maybe this needs to be in a new `it()` block. Too many it() blocks is often a problem, but this one is getting big, and that is also a problem. 

If there is shared setup between the it() blocks, that can be a shared function.

Reply:
Refactored